code/online_challenges/src/rosalind/complement_dna_strand/complement_Dna_Stands.py

```diff
@@ -0,0 +1,24 @@
+line = list(raw_input("Input DNA Strand: "))
```

Comment:
@RithvikKasarla I believe your branches have mixed. Since your other PR has no commit range, I would suggest you close your other PR and modify this PR to hold the corresponding information for DNA as well. (Since the changes are so small @AdiChat could merge this in one go).